packages/react-reconciler/src/ReactSubtreeTags.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+export type SubtreeTag = number;
+
+export const NoEffect = /*        */ 0b0000;
+export const DidBailout = /*      */ 0b0001;
+export const BeforeMutation = /*  */ 0b0010;
+export const Mutation = /*        */ 0b0100;
+export const Layout = /*          */ 0b1000;
```

Comment:
As a follow up, we should move the `Incomplete`, `ShouldCapture`, and `DidCapture` flags to this new field.